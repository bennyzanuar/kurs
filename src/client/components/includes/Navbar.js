import React, { Fragment } from 'react'
import PropTypes from 'prop-types'
import { withStyles } from '@material-ui/core/styles'
import AppBar from '@material-ui/core/AppBar'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'

const styles = {
    header: {
        backgroundColor: '#F66946'
    }
}

const NavBar = (props) => {
    const { classes } = props
    return(
        <>
            <AppBar position="static">
                <Toolbar className={classes.header}>
                    <Typography>
                        Exchange App
                    </Typography>
                </Toolbar>
            </AppBar>
        </>
    )
}
NavBar.propTypes = {
    classes: PropTypes.object.isRequired,
}

export default withStyles(styles)(NavBar)