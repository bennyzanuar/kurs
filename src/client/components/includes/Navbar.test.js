import React from 'react'
import { shallow } from 'enzyme'
import Navbar from './Navbar'

describe('<Navbar />', () => {
    it('should render navbar', () => {
        const component = shallow(<Navbar />)
        expect(component).toMatchSnapshot()
    })
})