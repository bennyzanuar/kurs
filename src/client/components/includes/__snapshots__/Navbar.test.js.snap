// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render navbar 1`] = `
<NavBar
  classes={
    Object {
      "header": "NavBar-header-1",
    }
  }
/>
`;
